std/hl/_std/haxe/io/Bytes.hx

```diff
@@ -171,6 +171,23 @@ class Bytes {
 	public static function ofData( b : BytesData ) : Bytes {
 		return new Bytes(b.bytes,b.length);
 	}
+	
+	public static function ofHex( s : String ) : Bytes {
+		var len = s.length;
+		if ( (len & 1) != 0 ) throw Error.OutsideBounds;
+		var l = len >> 1;
+		var b = new hl.Bytes(l);
+		for (i in  0...len)
+		{
+			var high = s.charCodeAt(i*2);
+			var low = s.charCodeAt(i*2 + 1);
+			high = (high & 0xf) + ( (high & 0x40) >> 6 ) * 9;
+			low = (low & 0xf) + ( (low & 0x40) >> 6 ) * 9;
+			b.setI32(i, ( (high << 4) | low)  & 0xff );
```

Comment:
"i" is an offset in bytes so maybe you want (i << 2) here, and you most likely need to allocate (l << 2) bytes and iterator from 0...l instead of 0...len